docs/concepts/typings.md

```diff
@@ -4,4 +4,78 @@ title: Type hints
+
+When using Strawberry to build graphQL APIs, as was shown in [Schema basics](https://strawberry.rocks/docs/general/schema-basics), type hints are required within classes decorated by `@strawberry.type` & `@strawberry.input` and functions decorated by `@strawberry.field` & `strawberry.mutation`. These type hints are sourced as the keywords `str`, `int`, `float`, and from packages imported directly from the Python standard libraries `typing`, that has been available since Python 3.5, `datetime`, and `decimal`.
+
+## Mapping to graphQL types
+
+The complete mapping of the required type hints for the relevant graphQL types is as follows:
+
+| GraphQL         | Python            |
+| --------------- | ----------------- |
+| `ID`            | `strawberry.ID`   |
+| `String`        | `str`             |
+| `Integer`       | `int`             |
+| `Float`         | `float`           |
+| `Array` or `[]` | `typing.List`     |
+| `Union`         | `typing.Union`    |
+| `Nullable`      | `typing.Optional` |
+| `Date`          | `datetime.date`   |
+| `Decimal`       | `decimal.Decimal` |
+
+where `typing`, `datetime`, and `decimal` are all part of the Python standard library.
+
+The usefulness of using `typing.List` instead of the keyword `list` is so that the types held in the list can be notated. A list of integers is notated as `typing.List[int]` and not as `list[int]`. The only effective equivalence that can be understood by the machine is that `typing.List[typing.Any]` is equivalent to `list`.
+
+The `typing.Optional` hint signifies that the variable being hinted can be its type or `None`. As a result, `Optional[int]` is equivalent to `Union[None, int]`.
+
+## Example
+
+A complete example of this, extending upon [Schema basics](https://strawberry.rocks/docs/general/schema-basics), might be the following:
+
+```python
+import datetime
+import typing
+import strawberry
+
+@strawberry.type
+class Book:
+    title: str
+    author: 'Author'
+    date_published: datetime.Date
+    price: decimal.Decimal
+    isbn_code: Union[int, str] # could be 1234567890000 or '123-4-56-789000-0'
+
+@strawberry.type
+class Author:
+    name: str
+
+    @strawberry.field
+    async def books(self, info: strawberry.types.Info) -> typing.List['Book']:
+        return await info.context['loaders'].load_books_by_author_name.load(self.name)
+
+@strawberry.type
+class Group:
+    name: Optional[str] # groups of authors don't necessarily need a name
+    authors: typing.List['Author']
+
+    @strawberry.field
+    async def books(self, info: strawberry.types.Info) -> typing.List['Book']:
+        names = [author.name for author in self.authors]
+        return await info.context['loaders'].load_books_by_author_names.load(names)
```

Comment:
can we make this full example work without having to import anything else? 😊

Reply:
Yes, I had a feeling that linking two pages like this is sub-optimal. Will change!